src/software/thunderscope/robot_diagnostics/drive_and_dribbler_widget.py

```diff
@@ -163,7 +157,10 @@ def setup_dribbler(self, title):
             self.dribbler_speed_rpm_slider,
             self.dribbler_speed_rpm_label,
         ) = common_widgets.create_slider(
-            "RPM", MIN_DRIBBLER_RPM * 1000, MAX_DRIBBLER_RPM * 1000, 1000
+            "RPM",
+            -self.constants.robot_max_speed_m_per_s * 1000,
+            self.constants.robot_max_speed_m_per_s * 1000,
```

Comment:
This slider represents the dribbler speed in rpm, so `self.constants.indefinite_dribbler_speed_rpm` should be used instead.